### ISSUE TITLE
Readme for generated directory tree

### ISSUE THREAD

**Issue by swistaczek** [dry-rb/dry-web-roda#17]
As new user I would like to read some introduction which learn me key concepts behind generated directory tree. Some simple example like super-simple todolist application will be very helpful for newbie.

**Comment by solnic** [comment]
We should probably just add user docs to dry-rb.org

**Comment by timriley** [comment]
Agree. @swistaczek, just FYI, docs are still a big to-do for this, but please reach out if you have any questions in the meantime.

**Comment by alejandrobabio** [comment]
Hi @swistaczek, 

A year after your original post, I guess you should be an expert in dry-web-roda or you stopped to use it at all. 

A month ago I shared a ToDo app built with dry-web-roda. It is available [here](https://github.com/alejandrobabio/drw-todo-example), comes with a [step by step document](http://dry-web-roda-todo-app.readthedocs.io/en/latest) and you can see it working [here](https://dwr-todo-example.herokuapp.com). I don't remember, but I think I took the idea from this issue because I was already liked your post.

In the document you will find:

* The folder structure of a dry-web-roda app
* DB migrations, repositories and relations
* Authentication
* Authorization
* The use of the Roda `flow` plugin to call `dry-transaction`s and its output processing
* Operations called in each step of the transactions
* Scoping data for the current user by policies scopes (idea was taken from Pundit)
* Input validation with  `dry-validation` 
* Showing the current user in layout
* Transactions for add and complete your tasks

If you need to use sprockets in a dry-web-roda app, perhaps you are migrating a Rails app (like me). I published my code in this [post](http://alejandrobabio.com/2017/12/11/dry-web-roda-and-sprockets.html).

Disclaimer: this is not an official document, it's only the way that I'm using this gem right now. Of course, I tried to follow the standards and the good practices that I learned reading the code of dry-rb, berg and rom-rb.

**Comment by swistaczek** [comment]
Hey @alejandrobabio!
Thanks for your work! I will definitely check out this application and give my feedback. Unfortunately, recently I have not written a larger application in Ruby for a long time, time has dashed me towards the scala and kotlin. It's time to check what's new in the "old" box :).
